.github/wiki/Tests.md

```diff
@@ -0,0 +1,135 @@
+# Tests
+
+## Running Tests
+This section is about testing your changes to React Native as a contributor. If you haven't yet, go through the steps to set up your development environment for [building projects with native code][env-setup]. If you are intested in [writing tests](#writing-tests) for a React Native app, proceed to the next section.
+
+[env-setup]: http://facebook.github.io/react-native/docs/getting-started
+
+
+### JavaScript Tests
+
+The simplest way to run the JavaScript test suite is by using the following command at the root of your React Native checkout:
+
+```bash
+npm test
```

Comment:
```suggestion
yarn test
```

Reply:
Actually, just noticed that yarn is missing from the setup instructions as well

Reply:
Yarn is not used in the Getting Started guide intentionally, as it was written around the same time Yarn was released and the additional steps to install it were deemed extraneous as npm is already bundled with node.

I think we can update the website to use yarn, at which point we can do the same here. After all, we use yarn in our CI over npm.

Reply:
Yeah that makes sense - the only risk is that new devs may run into some strange dependency issues with the difference in how deps are installed between yarn and npm 👍